dream_layer_backend/dream_layer_backend_utils/shared_workflow_parameters.py

```
@@ -0,0 +1,685 @@
+            print("Refiner is disabled, skipping...")
+            return workflow
+
+        refiner_model_map = {
+            'sdxl-1.0': 'sd_xl_refiner_1.0.safetensors',
+            'sdxl-0.9': 'sdxl_refiner_0.9.safetensors',
+            'flux': 'flux_refiner.safetensors',
+            'sdxl-turbo': 'sdxl_turbo_refiner.safetensors',
+            'none': None
+        }
+        refiner_model = refiner_data.get('refiner_model', 'none')
+        refiner_ckpt = refiner_model_map.get(refiner_model, None)
```

Ruff (0.11.9)

Use `refiner_model_map.get(refiner_model)` instead of `refiner_model_map.get(refiner_model, None)`

Replace `refiner_model_map.get(refiner_model, None)` with `refiner_model_map.get(refiner_model)`

(SIM910)